ui/common/src/socket.ts

```diff
@@ -18,11 +18,10 @@ interface MsgOut extends MsgBase {}
 interface MsgAck extends MsgOut {
   at: number;
 }
-type Send = (t: Tpe, d: Payload, o?: any) => void;
 
 interface Options {
   idle: boolean;
-  pingMaxLag: number; // time to wait for pong before resetting the connection
```

Comment:
rename `pingMaxLag` to `pongTimeout`, because `pingMaxLag` makes it sound like an observed statistic rather than a configuration variable